packages/gatsby/src/schema/infer-graphql-input-fields.js

```diff
@@ -205,6 +205,7 @@ export function inferInputObjectStructureFromNodes({
       prefix: `${prefix}${_.upperFirst(key)}`,
     })
 
+    if (typeof key !== 'string') return
```

Comment:
Yes that is what I am doing now ... 

I will keep you posted what he is printing out here. I can't see how this is working, this is still magic for me so need so help I suppose :)